tests/ext/xgboost/test_dataset.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import xgboost
+
+from ebonite.core.analyzer.dataset import DatasetAnalyzer
+from ebonite.ext.xgboost.dataset import DMatrixDatasetType
+
+
+@pytest.fixture
+def dtype(dmatrix):
+    return DatasetAnalyzer.analyze(dmatrix)
+
+
+def test_hook(dtype):
+    assert issubclass(dtype, DMatrixDatasetType)
+
+
+def test_serialize(dtype: DMatrixDatasetType()):
```

Comment:
what does `()` mean in this context?

Reply:
DMatrixDatasetType is a metaclass, kind of. So `()` means that we actually creating *type*